src/main/java/com/fasterxml/jackson/databind/ser/impl/BeanAsArraySerializer.java

```diff
@@ -214,10 +214,6 @@ protected final void serializeAsArray(Object bean, JsonGenerator gen, Serializer
                     prop.serializeAsElement(bean, gen, provider);
                 }
             }
-            // NOTE: any getters cannot be supported either
```

Comment:
... related to earlier, is "any getter" actually supported here? Does that work since "any getter" is now passed as `BeanPropertyWriter`?

Reply:
plz see my comment above